examples/client/Locomotion/src/context/settings/index.js

```diff
@@ -121,9 +121,13 @@ const useSettings = () => {
     setAppSettingsStates(appSettingsState);
   }, [appSettingsState]);
 
-  const loadShowPrice = async () => {
-    const hidePrice = await getSettingByKey(settingsKeys.HIDE_PRICE);
-    setShowPrice(!hidePrice);
+  const loadShowPrice = async (showPriceToMembersOfBA) => {
```

Comment:
lets extract it to a util resusable funcion that gets as params
{ loadShowPrice, getBusinessAccountById, businessAccountId } to reduce code duplication